doc/datamodel.md

```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
 In the following, the concepts presented in the above diagram are ellaborated.
 
 ### User
-This concept represents any end-user of the app and does not need to be 
+This concept represents any end-user of the app and does not need to be
 ellaborated. A user has a [TrainingJournal](#TrainingJournal).
 
+
+| Property Name | Type         | Remarks                                    |
+|----------------------|:-------------:|-------------------------------------------|
+| name                | `string`      |                                                    |
+| email                | `string`      |                                                    |
+| createdAt         | `datetime` |                                                    |
+| updatedAt        | `datetime` |                                                    |
+| TrainingJournal        | `TrainingJournal` |                                                    |

```

Comment:
the property name, e.g. name, always begins with a lower case letter, thus it is trainingJournal and not TrainingJournal.